transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/TransformTestWizardPage.java

```diff
@@ -484,12 +474,13 @@ private void doStatusUpdate() {
             }
         }
 
-        if (fPackageStatus.getCode() == StatusInfo.ERROR && (fPackageStatus.getMessage() == null
-                || fPackageStatus.getMessage().trim().isEmpty())) {
-            // override this particular case, since the default package is ok, though not great
-            fPackageStatus = new StatusInfo(NONE, null);
-        } else if (fPackageStatus.getCode() == StatusInfo.WARNING && fPackageStatus.getMessage() != null
-                && fPackageStatus.getMessage().contains("default package is discouraged")) { //$NON-NLS-1$
+        boolean defaultPkg = fPackageStatus.getCode() == StatusInfo.ERROR && (fPackageStatus.getMessage() == null
+            				 || fPackageStatus.getMessage().trim().isEmpty());
+        if (!defaultPkg) {
+        	defaultPkg = fPackageStatus.getCode() == StatusInfo.WARNING && fPackageStatus.getMessage() != null
+                		 && fPackageStatus.getMessage().contains("default package is discouraged"); //$NON-NLS-1$
```

Comment:
Although I hate the idea of referencing internal classes, this is definitely not the only place that it happens and I do worry about localization (though I don't believe we've had any issues reported yet with localization and this is far from the only place where it comes up), I think referencing the inner class is better than having a brittle string that could change without warning down the line. So that's a wishy washy way of giving a +1 for the internal class though I hate the idea.